README.md

```diff
@@ -88,19 +92,19 @@ jobs:
         run: "spin --version"
 ```
 
-## `fermyon/action/spin/push`
+## Push Spin app to a Registry - `fermyon/actions/spin/push`
 
-Build and push the `spin` app to Registry
+Build and push the `spin` app to your desired OCI Registry (note that this registry must have a publicly accessible endpoint). Also note this action has a prerequisite on Spin already being installed. 
 
 ### Inputs
 
-| Name               | Required | Description                                                                | Default   |
-| ------------------ | -------- | -------------------------------------------------------------------------- | --------- |
-| registry_reference | Required | Push the spin app to OCI Registry. e.g. ghcr.io/fermyon/cloud-start:v0.0.1 | -         |
-| manifest_file      | Optional | Path to `spin.toml`                                                        | spin.toml |
-| registry           | Optional | if provided, used to login to OCI Registry                                 | -         |
-| registry_username  | Optional | if provided, used to login to OCI Registry                                 | -         |
-| registry_password  | Optional | if provided, used to login to OCI Registry                                 | -         |
+| Name               | Required |  Default   |Description                                                                |
+| ------------------ | -------- | --------- | -------------------------------------------------------------------------- |
+| registry_reference | True | -         | The registry and reference for to publish the app to e.g. ghcr.io/fermyon/cloud-start:v0.0.1 |
+| manifest_file      | False | spin.toml | Path to `spin.toml`                                                        |
+| registry           | False | -         | if provided, used to login to OCI Registry                                 |
+| registry_username  | False | -         | if provided, used to login to OCI Registry                                 |
+| registry_password  | False | -         | if provided, used to login to OCI Registry                                 |
```

Comment:
I notice the example is showing `docker/login-action` rather than using this action's registry auth.  It would be good to show these fields in action (particularly with GHCR and a GH registry token, since this is a GH action).